libraries/AP_Proximity/AP_Proximity_RPLidarA2.cpp

```diff
@@ -394,7 +394,8 @@ void AP_Proximity_RPLidarA2::parse_response_data()
             Debug(2, "UART %02x %02x%02x %02x%02x", payload[0], payload[2], payload[1], payload[4], payload[3]); //show HEX values
             // check if valid SCAN packet: a valid packet starts with startbits which are complementary plus a checkbit in byte+1
             if ((payload.sensor_scan.startbit == !payload.sensor_scan.not_startbit) && payload.sensor_scan.checkbit) {
-                const float angle_deg = payload.sensor_scan.angle_q6/64.0f;
+                const float angle_dir = (frontend.get_orientation(state.instance) == 1) ? -1.0f : 1.0f;
```

Comment:
This should really switch on the orientation.  We don't have an enum for that, 'though, so *shrug*.

angle_sign might be better.

Reply:
txs, changed to angle_sign. I agree about how we should have an enum for orient but i'll leave that for another day